components/Login/SignIn.tsx

```diff
@@ -32,26 +36,44 @@ interface FormValues {
   checked: Array<string>
 }
 
+// checks if the user is able to log in
+export function login(data: Session | null, router: NextRouter) {
+  if (data?.user) {
+    if (data.user.id) {
+      router.push('/DynamicPOC')
+    }
+  }
+}
+
 // component for the signin page - includes form (and styling) for validation
 function SignIn() {
+  const router = useRouter()
```

Comment:
Is Route a hook? What exactly does it do?

Reply:
Next router just lets us push people around pages when we want to instead of right when the user clicks on something like a link